src/libraries/Microsoft.Extensions.Configuration.Abstractions/README.md

```diff
@@ -0,0 +1,13 @@
+# Microsoft.Extensions.Configuration.Abstractions
+
+`Microsoft.Extensions.Configuration.Abstractions` allows for implementing building different kinds of configuration providers to retrieve key/value pair configuration values from in the form of `IConfiguration`.
+
+Documentation can be found at https://learn.microsoft.com/en-us/dotnet/core/extensions/configuration.
+
+## Contribution Bar
+- [x] [We consider new features, new APIs, bug fixes, and performance changes](../../libraries/README.md#primary-bar)
+
+The APIs and functionality are mature, but do get extended occasionally.
+
+## Deployment
+[Microsoft.Extensions.Configuration.Abstractions](https://www.nuget.org/packages/Microsoft.Extensions.Configuration.Abstractions) is not included in the shared framework. The package is deployed as out-of-band (OOB) and needs to be installed into projects directly.
```

Comment:
>is not included in the shared framework. The package is deployed as out-of-band (OOB) and needs to be installed into projects directly. [](http://example.com/codeflow?start=130&length=135)

is included in the ASP.NET Core shared framework. The package is deployed as out-of-band (OOB) too and can be referenced into projects directly.